test/mock/service.go

```diff
@@ -23,12 +23,20 @@ import (
 
 // Mock values
 var (
-	HelloService                        = MakeService("hello.default.svc.cluster.local", "10.1.0.0")
-	WorldService                        = MakeService("world.default.svc.cluster.local", "10.2.0.0")
-	Discovery    model.ServiceDiscovery = &ServiceDiscovery{
+	HelloService   = MakeService("hello.default.svc.cluster.local", "10.1.0.0")
+	WorldService   = MakeService("world.default.svc.cluster.local", "10.2.0.0")
+	ExtHTTPService = MakeExternalHTTPService("httpbin.default.svc.cluster.local",
+		"httpbin.org", "10.3.0.0")
+	ExtHTTPSService = MakeExternalHTTPSService("httpsbin.default.svc.cluster.local",
```

Comment:
I left it in there originally because until external https services are supported - it should not appear in RDS/CDS/egress outputs.  And until they are supported, if someone pushes a change to the config generation logic - I wanted there to be something in place to catch it